Description: Use BFD on arm64, because the new binutils seems to be not able to link it anymore
Author: Gianfranco Costamagna <locutusofborg@debian.org>

Last-Update: 2017-07-02

--- ghc-8.0.2.orig/aclocal.m4
+++ ghc-8.0.2/aclocal.m4
@@ -594,7 +594,7 @@ AC_DEFUN([FPTOOLS_SET_C_LD_FLAGS],
         # On aarch64/linux and aarch64/android, tell gcc to link using the
         # gold linker.
         # Forcing LD to be ld.gold is done in FIND_LD m4 macro.
-        $3="$$3 -fuse-ld=gold -Wl,-z,noexecstack"
+        $3="$$3 -fuse-ld=bfd -Wl,-z,noexecstack"
         $4="$$4 -z noexecstack"
         ;;
 
@@ -2043,8 +2043,7 @@ AC_DEFUN([FIND_LLVM_PROG],[
 AC_DEFUN([FIND_LD],[
     FP_ARG_WITH_PATH_GNU_PROG([LD], [ld], [ld])
     case $target in
-        arm*linux*       | \
-        aarch64*linux*   )
+        arm*linux*)
             # Arm and Aarch64 requires use of the binutils ld.gold linker.
             # This case should catch at least arm-unknown-linux-gnueabihf,
             # arm-linux-androideabi, arm64-unknown-linux and
